requirements.txt

```diff
@@ -4,3 +4,5 @@ tqdm
 transformers
 accelerate
 peft>=0.3.0
+diffusers>=0.18.0
```

Comment:
Actually I think what is better to do is to put diffusers here: https://github.com/lvwerra/trl/blob/main/setup.py#L70 in the extra test dependency and make the dependency with diffusers optional, meaning we want to protect all diffusers related import with a new method `is_diffusers_available()`, similarly as peft: https://github.com/lvwerra/trl/blob/main/trl/import_utils.py#L24